src/components/image.js

```diff
@@ -9,8 +10,9 @@ const Image = ({ childImageSharp, extension, publicURL, alt, ...rest }) => {
         <img src={publicURL} alt={alt} />
```

Comment:
Also, can we use `<StaticImage>` here?
/cc @Tanuj22 @adithyaakrishna

Reply:
StaticImage doesn't support svgs.

Reply:
Yeah, I had tried using StaticImage but the svg files wont render